assets/index.html

```diff
@@ -141,6 +141,7 @@ <h1 id="main-head">
         <li><a href="/ja" class="button-link"><span class="flag">🇯🇵</span> 日本語</a></li>
         <li><a href="/it" class="button-link"><span class="flag">🇮🇹</span> Italiano</a></li>
         <li><a href="/fr" class="button-link"><span class="flag">🇫🇷</span> Français</a></li>
+        <li><a href="/es" class="button-link"><span class="flag">ES</span> Español</a></li>
```

Comment:
Here please use the corresponding flag emoji: 🇪🇸

Reply:
Emoji included. Many thanks!